examples/benchmark/benchmark_utils.py

```diff
@@ -0,0 +1,111 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  // SPDX-License-Identifier: BSD
+import time
+from dataclasses import dataclass
+
+import numpy as np
+import psutil
+import torch.cuda
+from pynvml import *
+
+monitor_gpu = False
+if torch.cuda.is_available():
+    monitor_gpu = True
+    nvmlInit()
+
+
+@dataclass
+class ExperimentResult:
```

Comment:
This class doesn't look like it's actually making use of the dataclass annotation?